diff --git a/node_modules/@nodert-win10-20h1/windows.ai.machinelearning/binding.gyp b/node_modules/@nodert-win10-20h1/windows.ai.machinelearning/binding.gyp
index 8c696a4..26bd707 100644
--- a/node_modules/@nodert-win10-20h1/windows.ai.machinelearning/binding.gyp
+++ b/node_modules/@nodert-win10-20h1/windows.ai.machinelearning/binding.gyp
@@ -72,7 +72,7 @@
     ],
     "msvs_settings": {
       "VCCLCompilerTool": {
-        "AdditionalOptions": ["/ZW"],
+        "AdditionalOptions": ["/ZW /bigobj /std:c++17"],
         "DisableSpecificWarnings": [4609]
       }
     }
