diff --git a/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/CollectionsWrap.h b/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/CollectionsWrap.h
index 3304539..e92ee6f 100644
--- a/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/CollectionsWrap.h
+++ b/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/CollectionsWrap.h
@@ -417,8 +417,9 @@ class IterableWrapper : NodeRT::WrapperBase {
 
   static void New(Nan::NAN_METHOD_ARGS_TYPE info) {
     NodeRT::Utils::SetHiddenValue(
-        info.This().Nan::New<String>("__winRtInstance__").ToLocalChecked(),
+        info.This(), Nan::New<String>("__winRtInstance__").ToLocalChecked(),
         True());
+
     info.GetReturnValue().Set(info.This());
   }
 
diff --git a/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/binding.gyp b/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/binding.gyp
index 7a8ea40..ca7244a 100644
--- a/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/binding.gyp
+++ b/node_modules/@nodert-win10-rs4/windows.ai.machinelearning.preview/binding.gyp
@@ -71,7 +71,7 @@
     ],
     "msvs_settings": {
       "VCCLCompilerTool": {
-        "AdditionalOptions": ["/ZW"],
+        "AdditionalOptions": ["/ZW /std:c++17"],
         "DisableSpecificWarnings": [4609]
       }
     }
